launcher/Application.cpp

```diff
@@ -285,12 +285,21 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
             return;
         }
     }
+
     m_instanceIdToLaunch = args["launch"].toString();
     m_serverToJoin = args["server"].toString();
     m_profileToUse = args["profile"].toString();
     m_liveCheck = args["alive"].toBool();
     m_zipToImport = args["import"].toUrl();
 
+    // error if --launch is missing with --server or --profile
+    if((!m_serverToJoin.isEmpty() || !m_profileToUse.isEmpty()) && m_instanceIdToLaunch.isEmpty())
+    {
+        std::cerr << "--server and --profile can only be used in combination with --launch!" << std::endl;
```

Comment:
did you mean --account

Reply:
Nope.
https://github.com/PolyMC/PolyMC/blob/6b4469c6cce517dabe967c12b30648e0c41f12d5/launcher/Application.cpp#L244-L247